TACC/Core-CMS
# fix(ecep): tup-151: hide "by: " line, stronger

## Overview / Changes

Update ECEP styles to hide byline for news on ECEP, for article **and** article list.

## Related

- requires https://github.com/TACC/Core-CMS-Resources/pull/155

## Testing

1. Load styles on page with news articles list.
2. Load styles on page with news article.
3. Neither page should show "by: …".

## Screenshots

| Article List | Article |
| - | - |
| ![Screen Shot 2022-07-25 at 10 19 23](https://user-images.githubusercontent.com/62723358/180814798-e3e4ddfa-92d3-4bbf-b2b7-a09b79f82927.png) | ![Screen Shot 2022-07-25 at 10 19 36](https://user-images.githubusercontent.com/62723358/180814812-0261a271-59d8-4b57-8411-16d14d89644f.png) |


## Notes

Client request, minimal code to fix. I moved to fast last time (ebfc9df, https://github.com/TACC/Core-CMS-Resources/pull/154).
